### PR TITLE
Adds a Space Gym Vault

### PR DESCRIPTION
**Pull request by Arthurdentist**
This PR adds a new vault to the game, the 'gym'.

gym ingame

The gym has two unique 'gym loot' spawners and a standard maint loot spawner. It also has a unique Darkholme suit named 'jabroni outfit' as well as a pill bottle full of creatine 'gym supplements' There is an issue with the pills which I need to resolve before it can be merged but that shouldn't be too tricky. The gym is protected by three unique zombies; Leatherman, Boss of this Gym and Boy Next Door. 

This isn't finished yet and i'm looking for suggestions for loot and other things. Its loot is pretty weak compared to the other vaults but im not sure what else to add, if anything.

### CODE REVIEW

**Comment by Kitfox88** [comment]
fucking amazing :100: :+1:

**Comment by ihadtoregisterforthis** [comment]
Two leather suits?

**Comment by Boxe** [comment]
but is it leatherman or leatherhead?

the answer is both

**Comment by unid15** [comment]
Make the leather****\* have a 50% chance of being named leatherman and 50% chance of being named leatherhead

**Comment by ghost** [comment]
It's leatherman though

**Comment by Boxe** [comment]
sounds like leathermeh to me

**Comment by Arthurdentist** [comment]
@ihadtoregisterforthis Just the one. You can't remove the one the zombie is 'wearing'.

**Comment by KingDairy** [comment]
can we get a dungeon master skull cap while we're at it

**Comment by ihadtoregisterforthis** [comment]
@Arthurdentist That's terrible

**Comment by TheEliminator16** [comment]
The real question is how far away is the leather club

**Comment by Arthurdentist** [comment]
@ihadtoregisterforthis there is no way I'm coding in the ability for zombies to wear clothes just for this. For now its just a unique icon.

@TheEliminator16 It's two blocks down.

**Comment by TheEliminator16** [comment]
> gay porn memes find their way into spessmen

I love this server

**Comment by ihadtoregisterforthis** [comment]
No homo

**Comment by Boxe** [comment]
@TheEliminator16

well most of the admins are gay so it's no surprise

**Comment by Arthurdentist** [comment]
Fixed the issue with the creatine pills and added two more loot spawners as well as a couple of decals to make it look more ruined. 

If you compare the two images you can see the advantage of using spawners like this. Using spawners you get 'variable' loot so that there is no guarantee that you'll necessarily get over powered goodies. In this instance only one loot holder spawned an SE and it was the 'fat' SE but if you get lucky you could find something nicer. 

gym updated

**Comment by Intigracy** [comment]
@Arthurdentist no need to code it (simple mobs wearing clothes) in specially, look at how russians handle it.

Short answer: It spawns a corpse when they die and equips it with the clothing the mob was wearing.

**Comment by Exxion** [comment]
This is a good label

**Comment by clusterfack** [comment]
Your memes end here

**Comment by FudgePucker** [comment]
:+1: 
Now all you got to do is add a leather club vault.

**Comment by Arthurdentist** [comment]
this is ready to merge unless someone really wants me to change how the leatherman zombie works. Suppose I could also changelog but I hear its broken.

**Comment by NigglyWiggly** [comment]
You should change the leatherman zombie to have the correct name

**Comment by KingDairy** [comment]
someone should just stream lords of the locker room so we can settle this in the ring

**Comment by PJB3005** [comment]
What the fuck is this shitty pretending to be [Honk] tag.

**Comment by Arthurdentist** [comment]
merge when

**Comment by 9600bauds** [comment]
It's not the Honk tag, it's for memes.
Honk does not accurately describe the content and does not have a stigma

**Comment by Arthurdentist** [comment]
does the memes tag have a stigma?

This is the only thing I have ever seen tagged with it.

**Comment by Exxion** [comment]
Because it's the only thing tagged with it

**Comment by Arthurdentist** [comment]
gee whizz

**Comment by TheEliminator16** [comment]
Really the "Leather***" mob needs to randomly pick "man" and "head".
You wouldn't want to inaccurately portray a very divisive issue, would you?

**Comment by Exxion** [comment]
> Cluster "NO FUN ALLOWED" Fack

Didn't you also remove the Usain Bolt label?

**Comment by clusterfack** [comment]
Labels aren't there to be fun or entertaining, they are meant to organize issues and PRs for coders to effectively find them so we can do our collaborating more easily.

It's the same reason I mock shitty bug reports that spend 80% of their text to make fun of coders or spout memes instead of reporting issues.

**Comment by Exxion** [comment]
But [Usain Bolt] was added _by Pomf himself_

Also explain [Honk] then

**Comment by Intigracy** [comment]
Honk is the one joke label that's permitted.

Fun is allowed some of the time, it just shouldn't be spamming the label list.

**Comment by TheEliminator16** [comment]
What about [OH GOD IT'S LOOSE]

File: maps/randomvaults/spacegym.dmm [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+"j" = (/turf/simulated/floor/airless,/area/vault/gym)
+"k" = (/obj/item/weapon/stool,/obj/item/weapon/reagent_containers/glass/rag,/turf/simulated/floor/airless,/area/vault/gym)
+"l" = (/obj/structure/closet/boxinggloves{name = "closet"},/obj/effect/decal/cleanable/cobweb2,/turf/simulated/floor/airless,/area/vault/gym)
+"m" = (/obj/machinery/shower,/turf/simulated/floor/airless,/area/vault/gym)
+"n" = (/obj/machinery/shower,/obj/item/weapon/storage/pill_bottle/creatine,/obj/effect/decal/cleanable/cobweb2,/turf/simulated/floor/airless,/area/vault/gym)
+"o" = (/obj/structure/closet,/turf/simulated/floor/airless,/area/vault/gym)
+"p" = (/obj/item/weapon/stool,/turf/simulated/floor/airless,/area/vault/gym)
+"q" = (/obj/map/spawner/misc/gym,/turf/simulated/floor/airless,/area/vault/gym)
+"r" = (/mob/living/simple_animal/hostile/necro/zombie{name = "boy next door"},/turf/simulated/floor/airless,/area/vault/gym)
+"s" = (/obj/item/clothing/under/darkholme{name = "jabroni outfit"},/turf/simulated/floor/airless,/area/vault/gym)
```
Comment (Probe1) [inline maps/randomvaults/spacegym.dmm:19]: :+1:  :100:

**Comment by Probe1** [comment]
also yeah what clusterfack said, don't go batty with labels please

**Comment by clusterfack** [comment]
@TheEliminator16 Those are high priority bugs

**Comment by Exxion** [comment]
Admittedly, [OH GOD IT'S LOOSE] is a serious tag with a silly name
But there's also [IT'S A FEAAAATUUUUREEEEEEEEE]

**Comment by Intigracy** [comment]
It'd serve the same purpose if it said "Feature" or "It's a Feature".

It's just made more witty. These all have actual USES outside of memes.

**Comment by TheEliminator16** [comment]
I actually didn't know that [OH GOD IT'S LOOSE] was meant to be serious, learn something new every day.

**Comment by 9600bauds** [comment]
> they are meant to organize issues

Someday someone will try to research the precedence of memes in the codebase and will be mildly inconvenienced

**Comment by Intigracy** [comment]
string up bauds by his bauds

**Comment by ihadtoregisterforthis** [comment]
HONK tag is about jokes, memes aren't a joke

**Comment by ihadtoregisterforthis** [comment]
https://youtu.be/yS8P4km0m3k

File: vgstation13.dme [hunk 2]
```diff
@@ -1932,6 +1932,7 @@
 #include "maps\defficiency\areas.dm"
 #include "maps\defficiency\pipes.dm"
 #include "maps\randomvaults\objects.dm"
+#include "maps\randomvaults\spacegym.dmm"
```
Comment (unid15) [inline vgstation13.dme:1935]: Shouldn't be here! This adds a 7th zlevel to the map

Comment (Intigracy) [inline vgstation13.dme]: the one above it too, I assume?

Comment (NigglyWiggly) [inline vgstation13.dme:1935]: Ruh roh

Comment (Intigracy) [inline vgstation13.dme:1935]: already fixed

Comment (NigglyWiggly) [inline vgstation13.dme:1935]: Then wouldn't it say something about it being an outdated version?
EDIT: Nevermind, just saw #8908

Comment (Intigracy) [inline vgstation13.dme:1935]: This was merged before it was fixed.

I made a seperate PR that fixed this right after reading unid's comment.

**Pull request merged**